lib/tasks/whitehall_importer.rb

```diff
@@ -190,6 +191,16 @@ def valid_state?(whitehall_edition)
       true
     end
 
+    def state_details(whitehall_edition, edition)
+      return nil unless whitehall_edition["state"] == "withdrawn"
```

Comment:
Don't need the nil here as it's implicitly returned - `return unless whitehall_edition["state"] == "withdrawn"`. Although I'm not sure you even need the return as you're raising an error before this code gets run checking for an unpublishing object and a state of withdrawn.